aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -1182,8 +1182,17 @@ define(["dojo/_base/declare",
             name: "alfresco/forms/controls/MaskingTextBox",
             config: {
                fieldId: "SHORTNAME",
-               mask: "[^0-9a-zA-Z\-\s]",
                targetId: "TITLE",
+               replacements: [
+                  {
```

Comment:
Good point. Didn't see that first time round, and it wasn't in the changes (obviously!).